package.json

```diff
@@ -27,28 +25,20 @@
          "properties": {
             "aks.draft.releaseTag": {
                "type": "string",
-               "default": "v0.0.22",
+               "default": "v0.0.25",
                "title": "Draft repository release tag",
                "description": "Release tag for the stable Draft tool release."
             }
          }
       },
       "commands": [
          {
-            "command": "aks-draft-extension.runDraftCreate",
-            "title": "Create Deployment Files"
+            "command": "aks-draft-extension.runDraftDockerfile",
+            "title": "AKS Developer: Draft a Dockerfile"
          },
          {
-            "command": "aks-draft-extension.runDraftSetupGH",
```

Comment:
Removes the webview experience commands. PMs thought it was too confusing for users.